README.md

```diff
@@ -69,3 +69,10 @@ To use the bot you need to write the following comment into a pull request:
 ```
 
 It takes a moment and then the bot should answer with a comment with more instructions on how to proceed.
+
+## Communication channels
+
+The Fellowship is using Matrix for communication. Right now there exists two channels:
+
+- [Polkadot Technical Fellowship Channel](https://matrix.to/#/#fellowship-members:parity.io): The channel for all Fellowship members to discuss. To get voice rights, you need to be part of the Fellowship. However, the channel is readable by anyone.
```

Comment:
```suggestion
- [Polkadot Technical Fellowship Channel](https://matrix.to/#/#fellowship-members:parity.io): The channel for all Fellowship members to discuss. To get voice rights, you need to be part of the Fellowship and of at least rank 1. However, the channel is readable by anyone.
```
Dont think most people realize what the word "member" means.

Reply:
I mean this is mainly for people from the outside that have no idea. I assume people that are a candidate will realize this.